vpp.conf.go

```diff
@@ -31,7 +31,7 @@ buffers {
 	buffers-per-numa 32768
 	# data-size was chosen using 4096 page - 256 bytes metadata - one cache line (64 bytes) to just *barely* fit in
 	# one page, since we can't split a buffer across a page.
-	default data-size 3776
+	default data-size %[2]d
```

Comment:
I feel it might be better to use go/templates

```go
unix {
  nodaemon
  log {{ .RootDir }}/var/log/vpp/vpp.log
  full-coredump
  cli-listen {{ .RootDir }}/var/run/vpp/cli.sock
  gid vpp
}
buffers {
	# buffers-per-numa was chosen as 256 buffers/interface * 128 possible interfaces
	buffers-per-numa 32768
	# data-size was chosen using 4096 page - 256 bytes metadata - one cache line (64 bytes) to just *barely* fit in
	# one page, since we can't split a buffer across a page.
	default data-size {{ .DataSize }}
}
## logging {
##   default-syslog-log-level debug
## }
api-trace {
## This stanza controls binary API tracing. Unless there is a very strong reason,
## please leave this feature enabled.
  on
## Additional parameters:
##
## To set the number of binary API trace records in the circular buffer, configure nitems
##
## nitems <nnn>
##
## To save the api message table decode tables, configure a filename. Results in /tmp/<filename>
## Very handy for understanding api message changes between versions, identifying missing
## plugins, and so forth.
##
## save-api-table <filename>
}
```

The usage could be
```go
var vppConfigBuilder strings.Builder
t := template.Must(template.New(“vppConfig”).Parse(vppConfContents))
err := t.Execute(configBuilder, struct{RootDir string, DataSize int}{rootDir, dataSize})
if err != nil {
	// TODO
}
//TODO: use vppConfig as string  via vppConfigBuilder.String()
```

The template usage could be super useful in future when we'll need to add more parameters into config ;)